src/daemon/dlt-daemon.c

```diff
@@ -1034,6 +1073,237 @@ int app_id_default_log_level_config_parser(DltDaemon *daemon,
 }
 #endif
 
+#ifdef DLT_TRACE_LOAD_CTRL_ENABLE
+static bool is_ascii_only(const char *str) {
+    while (*str) {
+        if ((unsigned char)*str > 127) {
+            return false;
+        }
+        str++;
+    }
+    return true;
+}
+
+/**
+ * Load configuration file parser
+ */
+int trace_load_config_file_parser(DltDaemon *daemon, DltDaemonLocal *daemon_local)
```

Comment:
This function should be defined at: dlt_config_file_parser for code cohesion

Reply:
fixed in new commit, will squash these once this is finalized